services/rook-ceph-cluster/1.14.5/defaults/cm.yaml

```
@@ -62,6 +62,7 @@ data:
       # priority classes to apply to ceph resources
       priorityClassNames:
         crashcollector: system-cluster-critical
+        exporter: system-node-critical
```

osd's and mgr's are the most important things and are needed for healthy quorum so maybe we can do this instead:

```suggestion
      priorityClassNames:
        all: system-cluster-critical
        osd: system-node-critical
        mgr: system-node-critical
```